GWorkspace/Desktop/Dock/Dock.m

```diff
@@ -834,7 +838,7 @@ - (NSDragOperation)draggingEntered:(id <NSDraggingInfo>)sender
         [icons removeObject: dndSourceIcon];
         [icons insertObject: dndSourceIcon atIndex: index];
         RELEASE (dndSourceIcon);
-        [self tile];  
+        [self  tile];  
```

Comment:
Curious as to the extra whitespace here.

Reply:
Typo. I'll fix it now.